test/API/endpoints/CloudRecording.spec.js

```diff
@@ -0,0 +1,219 @@
+const assert = require('assert');
+
+// import classes to test
+const API = require('../../../API');
+
+// import stub API
+const genStubAPIRequest = require('../helpers/stubAPIRequest');
+
+// import helper classes
+const utils = require('../../../EndpointCategory/helpers/utils');
+
+describe('Cloud Recording Endpoints', async function () {
+  let testAPI;
+  let defaultDate;
+  beforeEach(() => {
+    // Create API instance before every test
+    testAPI = new API({
+      key: 'fakeKey',
+      secret: 'fakeSecret',
+      sendZoomRequest: genStubAPIRequest(),
+    });
+
+    // Set up the default date value to use for comparision later
+    defaultDate = new Date();
+    defaultDate.setMonth(defaultDate.getMonth() - 6);
+    defaultDate = utils.formatDate(defaultDate);
+  });
+
+  it('lists Meeting Recordings', async function () {
+    const ret = await testAPI.cloudRecording
+      .listMeetingRecordings({ meetingId: 12345 });
+    assert.equal(
+      ret.path,
+      '/meetings/12345/recordings',
+      'path does not match'
+    );
+    assert.equal(
+      ret.method,
+      'GET',
+      'method does not match'
+    );
+    assert.deepEqual(
```

Comment:
You probably don't need to deepEqual here, maybe just an `assert(!ret.params)`, or normal equal if you want to make sure it's undefined. Afaik deep equal is only relevant for nested objects